tests/baselines/reference/quickInfoDisplayPartsInternalModuleAlias.baseline

```diff
@@ -516,7 +516,7 @@
     },
     "quickInfo": {
       "kind": "alias",
-      "kindModifiers": "export",
+      "kindModifiers": "export,export",
```

Comment:
@sandersn after my changes this baseline got modified, what are your thoughts about this?
Should I check for duplicates before pushing the kind modifiers from `resolvedSymbol`?
or maybe I should resolve the alias first `symbol = symbol.flags & SymbolFlags.Alias ? typeChecker.getAliasedSymbol(symbol) :  symbol ` and call `getNodeModifiers` just once

Reply:
Because I like seeing multiple kindModifiers in the rest of the baseline updates, I'd lean toward using a Set...except that we don't target ES6 yet, so you'll have to use `const modifiers = createMap<true>()` to mimic a set, then `arrayFrom(modifiers.keys())` to get the keys.

Resolving the alias first is going to mimic the old behaviour the closest, but I feel like it missed information compared to the baselines diffs in this PR.